docs/pdr_avhw.md

```diff
@@ -0,0 +1,619 @@
+# Flight Hardware
+
+Lead: Matthew Glazer
+
+[TOC]
+
+**Overview**: The scope of this PDR covers all hardware design and development as it currently stands for the Nucleus HAB launch. Development approaches, timelines and overall board information are provided. The design of payload, actuation mechanisms and software design are
+discussed in [Altitude Control Logic](pdr_altitudecontrol.md), [Mechanical Systems](placeholder.md) and [Flight Avionics Software](pdr_avsw.md), respectively.
+
+**Overview**: The Flight Hardware system for the nucleus mission provides all the power, computing and control elements required for mission success. There are seven primary PCBs and an assortment of secondary PCBs as part of this system.
+
+### Boards in work:
```

Comment:
Can probably cut this boards in work section. It was relevant for the review but isn't really relevant on the site and will get stale quick.

Reply:
removed